### PR TITLE
[Music]Prefer scraped album covers to embedded images

### PR DESCRIPTION
**Pull request by DaveTBlake**
Add an option to prefer scraped album covers over any images embedded in music files.

This is a long wanted feature that will help users automatically get the album art they want. While many users embed high resolution album covers their music files, and would not want the scraper to replace them, others have old poor quality art in there and would rather have any new pretty art that the scraper can fetch automatically appear. Previously avoiding using the embedded art meant selecting the album art from the list of scraped possibilities manually for each individual album (or of course re-tagging the music files)
___
Described in detail https://forum.kodi.tv/showthread.php?tid=331191

By default the album thumb (cover) art is automatically allocated in the following priority:
1) folder art, 
2) 1st embedded in music file, 
3) scraped
 This means that the scraped art only gets uses automatically when there is neither local folder art, nor art embedded in any of the music files.

It is right that local folder art, when it exists, always takes precedence, and this will continue to be the case. However with the new setting "Prefer online album art" enabled, the embedded art will only be used for album thumb when the scraper does not find art for that album.  When scraping remote sites, either when adding a music source,on library update or when clicking the "Refresh" button on the album information dialog, the first art found will replace any use of embedded art for the album.

### CODE REVIEW

**Comment by zag2me** [comment]
A hugely welcome option this one. Many times have people been confused about why different artwork has shown up. Thanks!

Logic looks good to.

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -12284,7 +12284,18 @@ msgctxt "#20223"
...
-#empty strings from id 20224 to 20239
+#: system/settings/settings.xml
+msgctxt "#20224"
+msgid "Prefer scraped album art"
+msgstr ""
+
+#. Description of setting with label #20224 "Prefer scraped album art"
+#: system/settings/settings.xml
+msgctxt "#20225"
+msgid "Downloaded album covers will be prefered over images extracted from embedded within a music file"
```
Comment (notspiff) [inline addons/resource.language.en_gb/resources/strings.po]: extract from what ?

Comment (DaveTBlake) [reply addons/resource.language.en_gb/resources/strings.po]: "... extracted from embedded within a music file"
But this obviously sounds wrong to you @notspiff, and actually it doesn't sound perfect to me either so any other suggestions?

Comment (MilhouseVH) [reply addons/resource.language.en_gb/resources/strings.po]:
```
msgid "Downloaded album covers will be prefered over embedded images extracted from within a music file" 
```
?

Comment (DaveTBlake) [reply addons/resource.language.en_gb/resources/strings.po]: Ah yes, I see it now, what was I on?
"Downloaded album covers will be preferred over those embedded within the music files"

**Comment by DaveTBlake** [comment]
Setting name and desc fixed (thanks for advice guys) - "Prefer online album art".

Looking to merge this at the weekend unless anyone objects.

**Pull request merged**